emscripten.py

```diff
@@ -1734,7 +1734,7 @@ def create_receiving(function_table_data, function_tables_defs, exported_impleme
   module_exports = exported_implemented_functions + function_tables(function_table_data)
   shared.Settings.MODULE_EXPORTS = [(f, f) for f in module_exports]
 
-  if not shared.Settings.SWAPPABLE_ASM_MODULE:
+  if shared.Settings.MINIMAL_RUNTIME or not shared.Settings.WASM_ASYNC_COMPILATION:
```

Comment:
how about setting `WASM_ASYNC_COMPILATION` when `MINIMAL_RUNTIME`?

Reply:
Strangely it would mean setting it 0.. since its want the same code here as the sync complication despite actually being async.

Reply:
I see, sounds ok then.